src/ol_infrastructure/applications/mitxpro/__main__.py

```diff
@@ -10,7 +10,7 @@
 from ol_infrastructure.lib.ol_types import Apps, AWSBase
 from ol_infrastructure.lib.pulumi_helper import parse_stack
 from ol_infrastructure.lib.stack_defaults import defaults
-from ol_infrastructure.lib.vault import mysql_sql_statements
+from ol_infrastructure.lib.vault import mysql_role_statements
```

Comment:
Should we keep `mitxpro` stuff out of this PR just to keep things strictly related to mitxonline?

Reply:
This is a change related to renaming a variable, so it's easier to keep it in since it's a small change.